### DISCUSSION TITLE
html template for license scanning

### DISCUSSION THREAD

**Discussion by fababs** [aquasecurity/trivy#5518] [Q&A]
### Question

Hi,

Is is possible to extend this  template: https://github.com/aquasecurity/trivy/blob/main/contrib/html.tpl

to handle license scanning?

Thanks!

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

**Comment by itaysk** [comment]
Hi there, we are not really maintaining additional output formats in Trivy anymore (except the basic ones) as this tends to be very subjective and customizable topic. The solution will be to use template or a plugin. We also have plans to make "output plugins" which will make discovering and using such custom formatters easier https://github.com/aquasecurity/trivy/issues/4860
